demo_modules/slideshow/fullscreen_display.js

```diff
@@ -154,8 +157,12 @@ class FullscreenClientDisplayStrategy extends interfaces.ClientDisplayStrategy {
             break;
           case 'full':
             content.style.display = 'block';
-            content.style.margin = '0px auto';
-            content.style.height = '100%';            
+            if (content.naturalWidth/this.surface.virtualRect.w >=
```

Comment:
why do you need this to be divided here instead of just checking if the image is wider than it is tall?